artiq/test/test_rpctool.py

```diff
@@ -0,0 +1,38 @@
+import os, sys
+import asyncio
+import unittest
+from artiq.protocols.pc_rpc import Server
+
+class Target:
+    def output_value(self):
+        return 4125380
+
+class rpctool_test(unittest.TestCase):
+    async def check_value(self):
+        proc = asyncio.create_subprocess_shell(
+                            sys.executable + " -m artiq.frontend.artiq_rpctool ::1 3249 call output_value", 
+                            stdout = asyncio.subprocess.PIPE,
+                            stderr = None)
+        result = await proc
```

Comment:
You could just do ``proc = await asyncio.create_subprocess_exec(...) ... await proc.communicate()...``

Reply:
I got this:  generator object has no attribute "communicate()"

Reply:
Forgot ``await``?